### PR TITLE
VVH review update

### PR DESCRIPTION
**Pull request by VVH**
changing the how to review prose so it reflects workflow.

### CODE REVIEW

File: static-pages/help-center/how-to-review.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ When a transcription is accurate and you do not need to make any changes, click
 A transcription that needs more work:
 </h2>
 <p>
-While reviewing, you can change a transcription to fix errors or add missing material. Remember, do not edit the spelling and grammar of the original document, but do correct any spelling errors or misreadings created by the transcriber. When you’re done, click the "Approve" button.
+While reviewing, you can change a transcription to fix errors or add missing material. Remember, do not edit the spelling and grammar of the original document, but do correct any spelling errors or misreadings created by the transcriber. When you’re done, click the "Save" and "Submit" buttons. The page will then need to be reviewed by another volunteer and will only be completed when a volunteer clicks "Approve" without making changes.
```
Comment (acdha) [inline static-pages/help-center/how-to-review.md]: The last half of the last sentence seems a bit complicated to read with with multiple conditional clauses — can we just remove it?

Comment (lalgee) [reply static-pages/help-center/how-to-review.md]: People are really confused about this so I think we need the clarification but can try to simplify.  How about:

When you’re done, click the "Save" and "Submit" buttons. Another volunteer will then need to review the page.  A page is complete when a reviewer clicks "Approve" without making changes.

Comment (rstorey) [reply static-pages/help-center/how-to-review.md]: The button reads "Accept" not "Approve"

**Comment by rstorey** [comment]
These changes need to be made in the fixture created to populate the "static page" content in the database.

**Comment by acdha** [comment]
Alternatively, we can copy-and-paste them into the production or staging database and let all future copy editing happen there.

**Comment by rstorey** [comment]
> Alternatively, we can copy-and-paste them into the production or staging database and let all future copy editing happen there.

But the .md files in this PR have already been deleted from master.

**Comment by acdha** [comment]
Yes — I was thinking we could just copy the contents out of the PR and close it after importing it into the database

**Comment by rstorey** [comment]
I moved the changes to the fixtures file so we won't have as many manual steps to get the latest updated content in there.

**Pull request merged**